bundle/config/resources/model_serving_endpoint.go

```diff
@@ -64,14 +64,10 @@ func (j *ModelServingEndpoint) ResourceDescription() ResourceDescription {
 		PluralName:            "model_serving_endpoints",
 		SingularTitle:         "Model Serving Endpoint",
 		PluralTitle:           "Model Serving Endpoints",
-		TerraformResourceName: "databricks_model_serving_endpoint",
+		TerraformResourceName: "databricks_model_serving",
```

Comment:
This one was incorrectly defined.

Reply:
Does it mean this resource does not work? or this field is not used?

Reply:
The field wasn't used before, but now it is. A test started to fail.